Actions/CreateApp/CreateApp.ps1

```diff
@@ -82,7 +82,12 @@ try {
     try {
         $settingsJsonFile = Join-Path $projectFolder $ALGoSettingsFile
         $SettingsJson = Get-Content $settingsJsonFile -Encoding UTF8 | ConvertFrom-Json
-        if (@($settingsJson.appFolders)+@($settingsJson.testFolders)) {
+        'appFolders','testFolders','bcptTestFolders' | ForEach-Object {
+            if ($settingsJson.PSObject.Properties.Name -ne $_) {
```

Comment:
Don't you mean `-notcontains` here? `-ne` will find all props that are not equal to the current one, which might always be the case.

```suggestion
            if ($settingsJson.PSObject.Properties.Name -notcontains $_) {
```

Reply:
You are right - I ment (-not (name -eq $_))